.github/workflows/ci.yml

```diff
@@ -464,6 +486,11 @@ jobs:
     timeout-minutes: 30
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # checkout all commits to be able to determine merge base for GIB
+      - name: Fetch base ref
+        run: |
+          [ "$GITHUB_REF_NAME" != master ] && git fetch --no-tags --prune --progress --no-recurse-submodules origin "$GITHUB_BASE_REF"
```

Comment:
Remove `--progress --no-recurse-submodules`

Reply:
Done! Sorry about this, I missed it during rebase.